app/policies/api_secret_policy.rb

```diff
@@ -1,6 +1,6 @@
 class ApiSecretPolicy < ApplicationPolicy
   def create?
-    true
+    !user_is_banned?
```

Comment:
**non-blocking:**

I think this more explicit, but either option is probably okay:
```suggestion
    true unless user_is_banned?
```

Reply:
Actually, I think I prefer your option since this result in return `nil` which is falsely, but probably not as good as returning a boolean in every case on a "huh" method. :laughing: So disregard this suggestion.

Reply:
Hmm I'm kind of indifferent to either, curious to see what others think though.

Reply:
I find it harder to read as "true unless user is banned" over "user is not banned" but that's just me and the way my brain interprets the `!` operator